### DISCUSSION TITLE
Confirm typo in Team class variables page?

### DISCUSSION THREAD

**Discussion by srednasykcin** [cwendt94/espn-api#408] [ESPN API] [answered]
@cwendt94  -- when looking at the variables for the Team class I noticed a typo in the "final_standings" variable. I was getting an error when looking for the final placing of a team and so I shortened the name to "final_standing" and it worked. Might want to edit that page? (https://github.com/cwendt94/espn-api/wiki/Team-Class-Basketball)

**Comment by cwendt94** [comment, marked as answer]
You are correct, thank you for catching this!
